### DISCUSSION TITLE
[Joke] Rewrite in rust

### DISCUSSION THREAD

**Discussion by Suya1671** [ThioJoe/YT-Spammer-Purge#264] [Ideas]
Hello, it seems you have written this app in python, however, the systems programming language known as "rust" is far superior to anything else that exists. C++? nope. JS? nope. Python? garbage. Rust? y e s

This is a joke pls dont kill me nor my fellow rust users we are more kind than you think :)

**Comment by kiatchun** [comment]


**Reply by Suya1671** [reply]
> > Hello, it seems you have written this app in python, however, the systems programming language known as "rust" is far superior to anything else that exists. C++? nope. JS? nope. Python? garbage. Rust? y e s
> 
> > 
> 
> > This is a joke pls dont kill me nor my fellow rust users we are more kind than you think :)
> 
> 
> 
> What about YOU rewrite the program in rust? y e s
> 
> 
> 
> This is a joke pls dont kill me nor my fellow non-rust users we are more kind than you think :)

Challenge (probably not) accepted.
If I start doing this Expect it to not go anywhere because I don't know pytho.

**Reply by Lumioze** [reply]
i would do this if the script was a bit easier to understand (it confuses me even though i'm probably more than up to the task because i know java and c++ n stuff)

**Reply by Suya1671** [reply]
> i would do this if the script was a bit easier to understand (it confuses me even though i'm probably more than up to the task because i know java and c++ n stuff)

Yeah the script confuses me. It's just 1 massive file.

**Comment by rbeer** [comment]
What's wrong with you? 😄

This isn't nearly as funny as it sounded in your head.

**Comment by Suya1671** [comment]
> What's wrong with you? 😄
> 
> This isn't nearly as funny as it sounded in your head.

What's wrong with me? My destroyed sense of humor. Yes I know it isn't that funny and I though for about 5 minutes before posting it
